elements/pfe-sass/functions/_custom-properties.scss

```diff
@@ -0,0 +1,134 @@
+/// Color
+///
+/// Returns CSS Var stack with exposed theme Var and respective SCSS fallback
+///
+/// @param {String} $cssvar - Variable name to be used and prepended with --pfe-theme
+/// @param {String} $category - Category name to be appended to variables within the map/system, default is "color"
+/// @param {Map} $map - SASS Map of variables
+/// @requires {String} $repo - Name of repo, which is "pfe"
+/// @use -
+///
+///     1.look up a color: /elements/pfe-sass/src/variables/_maps.scss
+///         (FYI, maps references: /elements/pfe-sass/src/variables/_colors.scss)
+///
+///     2. use like this:
+///         ```
+///         .my-element {
+///           color:       #{pfe-color(  surface--base--text  )};
+///           background:  #{pfe-color(  surface--base )};
+///         }
+///         ```
+
+@function pfe-color($cssvar, $category: 'color', $map: $pfe-colors) {
+    @if $category !="" {
+        $category: "#{$category}--";
+    }
+    // Start building the variable declaration
+    $var-declaration: "--#{$repo}-theme--#{$category}#{$cssvar}";
+    // If a fallback is defined, pass that in too
+    @if map-get($map, $cssvar) != null {
+        $var-declaration: "#{$var-declaration}, #{map-get($map, $cssvar)}";
+    }
+    // Return the variable declaration string
+    @return var(#{$var-declaration});
+}
+
+@function pfe-fallback($cssvar, $category: 'color', $map: $pfe-colors) {
```

Comment:
```suggestion
/// For use within other functions, not intended for direct usage. Please use pfe-local, pfe-apply-broadcast, or pfe-color
@function pfe-fallback($cssvar, $category: 'color', $map: $pfe-colors) {
```

Reply:
I'm not sure this is quite accurate.  You could use it in situations where you just want the color with no hook for theme or broadcast variables.